packages/cbioportal-utils/src/signal/GeneFrequencySummary.ts

```diff
@@ -1,5 +1,3 @@
-// this component comes from signal
```

Comment:
I think this belongs to model. Let's move this under model and rename it `SignalGeneFrequencySummary`? Also prefix the model names with Signal to make it clear, like `ISignalGeneFrequencySummary`, `ISignalTumorTypeFrequencySummary`, etc?

Reply:
@onursumer Do you think it makes sense to rename all signal related model with a `signal` prefix? For example https://github.com/cBioPortal/cbioportal-frontend/pull/3489/files#diff-a4120f05d3b6d91af87296f694be727c063beab1990c7c317cd66da0662ba43bR1